lib/trino-orc/src/main/java/io/trino/orc/metadata/statistics/StringStatisticsBuilder.java

```diff
@@ -170,4 +195,73 @@ private Slice dropStringMinMaxIfNecessary(Slice minOrMax)
         }
         return Slices.copyOf(minOrMax);
     }
+
+    static class StringCompactor
+    {
+        public static byte[] computeMin(byte[] bytes, int maxBytes)
+        {
+            if (maxBytes >= bytes.length) {
+                return bytes;
```

Comment:
The code is invoked only for truncation code path, so this line is dead code.
Let's rename the method to `truncateMin` and add `checkArgument(bytes.length > maxBytes)`